schema.sql

```diff
@@ -1,8 +1,10 @@
+-- Create table animals
 CREATE TABLE animals (
   id INT,
   name VARCHAR(255),
   date_of_birth DATE,
   escape_attempts INT,
   neutered BOOLEAN,
-  weight_kg DECIMAL(10,2)
+  weight_kg DECIMAL(10,2),
+  species VARCHAR(255)
```

Comment:
- [ ] Great effort with this update :clap: but really this is not practicable in real life because you will not risk destroying an entire database all because of a new field/entity/entry that you need to add/append/update, and there are some `SQL` commands/syntax that can really make this possible. I'd suggest that you read about the use of `ALTER` command/query to do this. It will look like this:
`   ALTER TABLE animals ADD species varchar(255);`

Reply:
@AbayomiOlaoye 

I totally agree with you 100% and that what actually did when I was practicing. :100: :ok_hand: 

but I was afraid of submit `alter` at my code to review and it considered wrong, :face_with_spiral_eyes:  so I decided to go with the safe option. :smile:  

however, I'll use `Alter` and change my sql code . :+1: